python/pyspark/ml/tests.py

```diff
@@ -1027,6 +1027,21 @@ def test_storage_levels(self):
         self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
         self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
 
+    def test_unknown_strategy(self):
```

Comment:
I think its probably good habit to add tests for new complex params we add (but probably not worth it to go back and fill it in). I know that as a user when the docs don't explain stuff closely enough turning to the tests is often a good fall back (so we can think of it as also helping with documentation in a way).